test/e2e/charts/clickhouse-tls/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
I am confused, why don't we make a more general chart that also replaces build/yamls/flow-visibility-e2e.yml?
This file is also only used for testing, and there is no reason to keep both

Reply:
I forgot to combine `build/yamls/flow-visibility-e2e.yml`. I've updated the `test/e2e/charts` folder.